atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CqlKeyValueService.java

```diff
@@ -1198,36 +1184,7 @@ String getFullTableName(TableReference tableRef) {
 
     @Override
     public void compactInternally(TableReference tableRef) {
-        Preconditions.checkArgument(!Strings.isNullOrEmpty(tableRef.getQualifiedName()),
-                "tableName:[%s] should not be null or empty", tableRef);
-        if (!compactionManager.isPresent()) {
-            log.error("No compaction client was configured, but compact was called."
-                    + " If you actually want to clear deleted data immediately"
-                    + " from Cassandra, lower your gc_grace_seconds setting and"
-                    + " run `nodetool compact {} {}`.", config.getKeyspaceOrThrow(), tableRef);
-            return;
-        }
-
-        long compactionTimeoutSeconds = config.jmx().get().compactionTimeoutSeconds();
-        try {
-            alterTableForCompaction(tableRef, 0, 0.0f);
-            CqlKeyValueServices.waitForSchemaVersionsToCoalesce("setting up tables for compaction", this);
-            compactionManager.get().performTombstoneCompaction(
-                    compactionTimeoutSeconds,
-                    config.getKeyspaceOrThrow(),
-                    tableRef);
-        } catch (TimeoutException e) {
-            log.error("Compaction could not finish in {} seconds. {}", compactionTimeoutSeconds, e.getMessage());
-            log.error("Compaction status: {}", compactionManager.get().getCompactionStatus());
-        } catch (InterruptedException e) {
-            log.error("Compaction for {}.{} was interrupted.", config.getKeyspaceOrThrow(), tableRef);
-        } finally {
-            alterTableForCompaction(
-                    tableRef,
-                    config.gcGraceSeconds(),
-                    CassandraConstants.TOMBSTONE_THRESHOLD_RATIO);
-            CqlKeyValueServices.waitForSchemaVersionsToCoalesce("setting up tables post-compaction", this);
-        }
+        /* no-op */
```

Comment:
should also override `shouldManuallyCompact`.

Reply:
It is by default false, right? That's what we want for Cassandra based KVSes, reading the Javadoc (though the method name is a bit awkward I think)

Reply:
Yep, this comment is from before we decided to switch the default from true to false, so now there's nothing to do :tada: